crates/ide/src/syntax_highlighting/inject.rs

```diff
@@ -258,9 +258,8 @@ fn module_def_to_hl_tag(def: Definition) -> HlTag {
         Definition::SelfType(_) => SymbolKind::Impl,
         Definition::Local(_) => SymbolKind::Local,
         Definition::GenericParam(gp) => match gp {
-            hir::GenericParam::TypeParam(_) => SymbolKind::TypeParam,
+            hir::GenericParam::TypeOrConstParam(_) => SymbolKind::TypeParam,
             hir::GenericParam::LifetimeParam(_) => SymbolKind::LifetimeParam,
-            hir::GenericParam::ConstParam(_) => SymbolKind::ConstParam,
         },
```

Comment:
This is incomplete, `SymbolKind::ConstParam` is gone now.

Reply:
Let's add a test for it. There are some tests to highlight const generics but they are not affected by this.

Reply:
I added a test for generics in doctests but it still doesn't hit there. Since this function is only used in doctests, maybe that part is actually unreachable?

Reply:
It might be?

Reply:
So anything else to do?